opf/anomaly_benchmark.py

```diff
@@ -1,12 +1,128 @@
 #!/usr/bin/env python
-import anomalyTester
-import modelParamsRDSE as mp
-# import modelParams as mp
 
+try:
+  import matplotlib.pyplot as plt
+except:
+  print "Failed to import matplotlib, plotting will not work."
+from collections import deque
+import os.path
+import csv
+try:
+  from nupic.algorithms import anomaly_likelihood as al
+  from nupic.frameworks.opf.modelfactory import ModelFactory
+except:
+  print "Cannot find nupic library!"
+  raise
+
+"""
+Anomaly Tester:
+The anomaly tester is a class that encapsulates opf codes. It makes handling with the opf easier.
+All you need to do is set the input csv file with `set_input()`, set the output csv file with `et_output()` and run
+the model with `run_model()`.
+
+If you intend to train HTM on an anomaly free data set first and then test the model on anomalous data,
+proceed the training as usual and then just reset the input file and run the model again.
+
+Example is shown in __main__.
+
+"""
+
+
+
+
+
+# GENERAL PRESET
+
+HEADERS = ["timestamp", "function", "phase", "has_anomaly"]
+
+
+class AnomalyTester:
+    def __init__(self, model_parameters, plotting=False):
+        self.plotting = plotting
+        self.reader = None
+        self.writer = None
+        self.al_helper = al.AnomalyLikelihood()
+        self.model = None
+        self.model = ModelFactory.create(model_parameters)
+
+        self.model.enableInference({'predictedField': 'function'})
+
+        if plotting:
+            raise NotImplementedError('Not yet implemented')
+# plt.ion()
+# fig = plt.figure()
+#
+# plt.title('Testing the model')
+# plt.xlabel('time [s]')
+# plt.ylabel('Y axe')
```

Comment:
removed the `model.py` as it was duplicit to anomalyTester, just copied the plot functionality comments, do we want it? (it does not seem to be working)